crates/firewheel-core/src/node.rs

```diff
@@ -84,15 +86,22 @@ impl AudioNodeInfo {
         self.call_update_method = call_update_method;
         self
     }
+
+    /// Custom `!Send` state that can be stored in the Firewheel context.
+    pub fn custom_state(mut self, custom_state: Option<Box<dyn Any>>) -> Self {
```

Comment:
Like with the `node_state` method on the audio context, could we make this generic over the state for some minor ergonomic gains?

```rs
pub fn custom_state<T: 'static>(mut self, custom_state: T) -> Self {
    self.custom_state = Some(Box::new(custom_state));
    self
}
```